modules/framework/galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/internal/streams/StreamsServiceImpl.java

```diff
@@ -73,7 +73,7 @@ private Stream createStreamFromProperties(String streamName, Map<String, String>
                     streamBean.setDescription(value);
                     break;
                 case "obr":
-                    streamBean.setObrLocation(value);
+                    streamBean.setObrs(value);
```

Comment:
Could this be called setObrsFromCommaSeparatedList

Reply:
Addressed in https://github.com/galasa-dev/galasa/pull/236/commits/a9ef197a47c1c7da97f4c1566f80ca24bce91184